src/main/java/net/masterthought/cucumber/json/deserializers/EmbeddingDeserializer.java

```diff
@@ -24,7 +24,16 @@
     @Override
     public Embedding deserialize(JsonNode rootNode, Configuration configuration) {
         String data = rootNode.get("data").asText();
-        String mimeType = rootNode.get("mime_type").asText();
+        JsonNode media = rootNode.get("media");
```

Comment:
extract `media` into static attribute

Reply:
I'm not sure what you mean by this, but have extracted the logic into an instance method.  It's not a static method because none of the other deserialisers have static helper methods.